tests/console/validate_multipath.pm

```diff
@@ -0,0 +1,172 @@
+# SUSE's openQA tests
+#
+# Copyright © 2020 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved. This file is offered as-is,
+# without any warranty.
+
+# Summary: Validate multipathing on the installed system.
+# The multipathing support in SUSE Linux Enterprise Server is based on
+# the Device Mapper Multipath module of the Linux kernel and the multipath-tools user space package.
+# the Multiple Devices Administration utility (multipath) can be used to view the status of multipathed devices.
+#
+# Maintainer: QA SLE YaST team <qa-sle-yast@suse.de>
+
+use strict;
+use warnings;
+use base "opensusebasetest";
+use testapi;
+use scheduler 'get_test_suite_data';
+use Test::Assert ':all';
+
+sub verify_packages_installed {
```

Comment:
I prefer the simplicity of this method, I don't need to check deeper if it is a pattern or is available or in which repo, but if there is some advantage I could adapt it.

Reply:
Then we will have to maintain the same functionality in two different places.

Reply:
for simple checking I use rpm, if I would choose zypper then I could reuse the function. I'm not using zypper because I don't foreseen expansion of this. It is not the same code from the point of view of maintenance meaning is not repeated. any other advantage?